doc-content/enterprise-only/openshift/truststore-template-proc.adoc

```diff
@@ -0,0 +1,82 @@
+[id='truststore-template-proc_{context}']
+= (Optional) Providing a truststore for accessing HTTPS servers with self-signed certificates
+:truststore_bc!:
+ifeval::["{context}"=="openshift-authoring"]
+:truststore_bc: {CENTRAL}
+endif::[]
+ifeval::["{context}"=="openshift-authoring-managed"]
+:truststore_bc: {CENTRAL}
+endif::[]
+ifdef::PAM[]
+ifeval::["{context}"=="openshift-managed"]
+:truststore_bc: {CENTRAL} Monitoring
+endif::[]
+ifeval::["{context}"=="openshift-freeform"]
+:truststore_bc: {CENTRAL} Monitoring
+endif::[]
+ifeval::["{context}"=="openshift-immutable"]
+:truststore_bc: {CENTRAL} Monitoring
+endif::[]
+endif::PAM[]
+
+Components of your {PRODUCT} infrastructure might need to use HTTPS access to servers that have a self-signed HTTPS certificate. For example,
+ifdef::truststore_bc[]
+{truststore_bc} and
+endif::truststore_bc[]
+{KIE_SERVER} might need to interact with an internal Nexus repository that uses a self-signed HTTPS server certificate.
+
+In this case, to ensure that HTTPS connections complete successfully, you must provide client certificates for these services using a truststore.
+
+Skip this procedure if you do not need {PRODUCT} components to communicate with servers that use self-signed HTTPS server certificates.
+
+.Procedure
+
+. Prepare a truststore with the certificates. Use the following command to create a truststore or to add a certificate to an existing truststore. Add all the necessary certificates to one truststore.
++
+[subs="attributes,verbatim,macros,quotes"]
+----
+keytool -importcert -file _certificate-file_ -alias _alias_ -keyalg _algorithm_ -keysize _size_ -trustcacerts -noprompt -storetype JKS -keypass _truststore-password_ -storepass _truststore-password_ -keystore _keystore-file_
```

Comment:
@mramendi Just a heads-up that the CCS-wide supplemental style guide that's coming soon is leaning toward variables like _certificate-file_ being ital like this (per IBM SG), but additionally all caps for clearer distinction, as in _CERTIFICATE-FILE_. Don't need to do anything right now, since the decision is still pending, but just a heads-up. I've started following that in Kogito just to get ahead of the curve, since it looks like that's going to happen. Anyway, just a note.

Reply:
Thanks! Won't change right now as the change would need to be made across the doc set.